templates/packages/order_edit.html

```diff
@@ -1,261 +1,199 @@
-                            </div>
-                            <div class="col">
-                                <h5 class="card-title">{{package.destination}}</h5>
-                            </div>
-                        </div>
-                        <ul class="card-subtitle list-inline text-muted">
-                            <li class="list-inline-item"><h6>{{package.origin.city}}, {{package.origin.street}}</h6></li>
-                            <li class="list-inline-item"><span class="bi-arrow-return-right"></span></li>
-                        </ul>
-                        <p class="card-text">
-                            <a class="btn btn-outline-primary mb-1 ms-1" href="/partner/{{partner.name}}/{{order.id}}/{{package.id}}/">Edit</a> <span class="text-{{package.status_color}}">{{package.status}}</span>
-                            <span class="row justify-content-end ms-1 fw-light mb-2">{{package.full_name}} {{package.phone_number}}</span>
-                            {% if is_staff%}
-                                <ul class="list-inline">
-                                    <li class="list-inline-item">
-                                        <form method="post" action="/partner/{{partner.name}}/{{order.id}}/update/"> {% csrf_token %}
-                                            <input type="hidden" name="package" value="{{package.id}}"/>
-                                            <input type="hidden" name="update-type" value="increase"/>
-                                            <button type="submit" class="btn btn-outline-success">Update</button>
-                                        </form>
-                                    </li>
-                                    <li class="list-inline-item me-4">
-                                        <form method="post" action="/partner/{{partner.name}}/{{order.id}}/update/"> {% csrf_token %}
-                                            <input type="hidden" name="package" value="{{package.id}}" />
-                                            <input type="hidden" name="update-type" value="revert"/>
-                                            <button class="btn btn-outline-danger">Revert</button>
-                                        </form>
-                                    </li>
-                                </ul>
-                            {%endif%}
-                        </p>
+</div>
+{%if has_unsaved_progress%}
+<div class="">
+    <h6 class="text-danger">*קיימים שינויים לא שמורים</h6>
+</div>
+{%endif%}
+<div class="row justify-content-center">
+    {% for package in packages%}
+    <div class="col-xxl-3 col-xl-4 col-lg-4 col-md-6 col-sm-6 mb-3">
+        <div class="card h-100">
+            <div class="card-body text-start">
+                <div class="row">
+                    <div class="col-auto text-decoration-none">
+                        <a type="button" data-bs-toggle="popover" data-bs-placement="bottom"

```

Comment:
So this seems broken

![image](https://user-images.githubusercontent.com/17641355/112116564-16109080-8bc3-11eb-95be-85b86e0d2478.png)

Reply:
fuck my final changes weren't committed, that's why it was broken and the thing was still in base

Reply:
It's still broken